e2e/tests/components/input-dropdown/input-dropdown.po.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { browser, by, element } from 'protractor';
+
+export class InputDropdownPage {
+
+    async getPage(): Promise<void> {
+        await browser.get('#/input-dropdown');
+    }
+
+    dropdown = element(by.id('dropdown'));
+    checkboxDisabled = element(by.id('checkbox2'));
+
+    confirmDropdownIsExpanded() {
```

Comment:
```suggestion
    isDropdownExpanded(): boolean {
```
This and all other functions here are missing return types.